frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx

```diff
@@ -60,7 +63,18 @@ export function FunnelTab(): JSX.Element {
                                 </h4>
                                 {clickhouseFeaturesEnabled && (
                                     <Row align="middle" style={{ padding: '0 4px' }}>
-                                        <span className="l5 text-muted-alt">
+                                        {showGroupsOptions && (
```

Comment:
This errors out on correlation analysis, since we don't yet support correlations for groups. (screams at joining from UUID -> string)

Reply:
Add to `correlationAnalysisAvailable` in `funnelLogic` ?